BrentOzarULTD/SQL-Server-First-Responder-Kit
# Blitz_673

Fixes #673 .

Changes proposed in this pull request:
 - Add code to Check 28: 'MSDB' AS DatabaseName ,
 - Add code to Check 29: 'model' AS DatabaseName ,
 - 

How to test this code:
 - Make sure a user table is in both MSDB and model
 - 
 - 

Has been tested on (remove any that don't apply):
 - Case-sensitive SQL Server instance
 - SQL Server 2008
 - SQL Server 2008 R2
 - SQL Server 2012
 - SQL Server 2014
 - SQL Server 2016
